test/testrunner.py

```diff
@@ -160,14 +162,29 @@ def main():
         if specific_test_type == "sanity":
             pull_dlc_images(all_image_list)
         if specific_test_type == "eks":
-            new_eks_cluster_name = setup_eks_clusters(dlc_images)
-        # Execute dlc_tests pytest command
-        pytest_cmd = ["-s", "-rA", test_path, f"--junitxml={report}", "-n=auto"]
+            frameworks_in_images = [framework for framework in ("mxnet", "pytorch", "tensorflow")
+                                    if framework in dlc_images]
+            if len(frameworks_in_images) != 1:
+                raise ValueError(
+                    f"All images in dlc_images must be of a single framework for EKS tests.\n"
+                    f"Instead seeing {frameworks_in_images} frameworks."
+                )
+            framework = frameworks_in_images[0]
+            eks_cluster_name = setup_eks_clusters(dlc_images)
```

Comment:
This will likely not work anymore now that you've changed setup_eks_clusters.

Reply:
Pushed the wrong commit. Correcting it now.